opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -197,13 +201,15 @@ def create_time_series_list(self, v_data, option_resource_type,
                                 metric_prefix):
         """ Create the TimeSeries object based on the view data
         """
-        series = monitoring_v3.types.TimeSeries()
-        series.metric.type = namespaced_view_name(v_data.view.name,
-                                                  metric_prefix)
-        set_monitored_resource(series, option_resource_type)
-
+        time_series_list = []
         tag_agg = v_data.tag_value_aggregation_data_map
         for tag_value, agg in tag_agg.items():
+            series = monitoring_v3.types.TimeSeries()
+            series.metric.type = namespaced_view_name(v_data.view.name,
+                                                      metric_prefix)
+            set_metric_labels(series, v_data.view, tag_value)
```

Comment:
@mayurkale22 this looks like a bug to me, is the right behavior to set a label for each tag value (not just `tag_value`) for each series or only set a single label (in which case `set_metric_label` needs to change).